docs/articles/documentation/using-testcafe/common-concepts/browsers/testing-in-headless-mode.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: docs
+title: Testing in Headless Mode
+permalink: /documentation/using-testcafe/common-concepts/browsers/testing-in-headless-mode.html
+---
+# Testing in Headless Mode
+
+TestCafe allows you to run tests in Google Chrome and Mozilla Firefox without any visible UI shell - in the headless mode ([Chrome Headless](https://developers.google.com/web/updates/2017/04/headless-chrome), [Firefox Headless](https://developer.mozilla.org/en-US/Firefox/Headless_mode)). Use the `headless` browser parameter to launch a browser in the headless mode.
+
+```sh
+testcafe "chrome:headless" tests/sample-fixture.js
+```
+
+```js
+runner
+    .src('tests/sample-fixture.js')
+    .browsers('chrome:headless')
+    .run();
+```
+
+Specify a path to the browser executable if you use a portable version of the browser:
+
+```sh
+testcafe "firefox:path/to/firefox:headless" tests/sample-fixture.js
+```
+
+```js
+runner
+    .src('tests/sample-fixture.js')
+    .browsers('firefox:path/to/firefox:headless')
+    .run();
+```
+
+Note that the [Take screenshot](../../../test-api/actions/take-screenshot.md) and [Resize window](../../../test-api/actions/resize-window.md) features are fully supported in the headless mode.
+
+> In the headless mode, the browser's viewport size is `1280x800` by default. You can change it via the [ResizeWindow](../../../test-api/actions/resize-window.md) command or via [Device Emulation Mode](using-chrome-device-emulation.md#emulator-parameters) parameters (for Chrome).
+
+## CDP Port
```

Comment:
There is similar configuration option for Firefox called `marionettePort`. It's almost totally the same, but it's better to describe it like `remote control/automation port`. https://developer.mozilla.org/ru/docs/Marionette